### PR TITLE
🎩 Update mobile menu

### LINKED ISSUE #197: Update mobile menu and navbar
1. Issues with scrolling - menu only visible when actively scrolling, and hides when at the top. Instead, make it static at the top and always visible, but fixed above the home image at all times.
2. Add the YMCA logo on the mobile nav bar (no need for search icon yet)
3. Have the menu pull out from the right and style it like the image - no need to add the resource categories in yet though

image
image

### PR DESCRIPTION
**Pull request by kenetec**
* Adds YMCA icon to mobile nav.
* Restyles mobile menu.

Closes #197

### CODE REVIEW

File: client/src/components/mobile/NavigationMobile.js [hunk 1]
```diff
@@ -3,14 +3,15 @@
...
+import { Drawer, Button, Icon, Menu } from 'antd';
 
 type Props = {
   authed: Boolean,
+  authRoleIsEquivalentTo: String => Boolean,
 };
 
 const NavMobile = (props: Props) => {
-  const { authed } = props;
+  const { authed, authRoleIsEquivalentTo } = props;
```
Comment (jackieo5023) [inline client/src/components/mobile/NavigationMobile.js:14]: I can't tell for sure, but is this unused?  If so we should remove it :P

File: client/src/components/mobile/NavigationMobile.js [hunk 2]
```diff
@@ -28,68 +29,93 @@ const NavMobile = (props: Props) => {
...
+            >
+              <NavLink className="nav-mobile-option" exact to="/resources">
+                Resources
               </NavLink>
-            </p>
-          ) : (
-            <p>
-              <NavLink className="nav-mobile-option" to="/logout">
-                Logout
+            </Menu.SubMenu> */}
```
Comment (jackieo5023) [inline client/src/components/mobile/NavigationMobile.js:91]: Can we remove this comment?

File: client/src/css_mobile/Navigation.css [hunk 3]
```diff
@@ -7,15 +7,49 @@
...
   transition: color 0.3s ease;
-  list-style-type: none;
+  font-size: 1.2em !important;
+  text-align: left !important;
 }
 
 .nav-mobile-option:hover {
-  color: tomato;
-  text-decoration: none;
+  color: tomato !important;
```
Comment (jackieo5023) [inline client/src/css_mobile/Navigation.css:53]: Can we make the styling similar to the desktop navbar, where if you hover it it's a bolded black, and when its selected it's bolded black as well?

File: client/src/components/mobile/NavigationMobile.js (hunk 1 above)
Comment (kenetec) [reply client/src/components/mobile/NavigationMobile.js:14]: It will used to determine whether to show the NavLinks to admin and user page.

**Pull request merged**